MC:13321 - Updated docs for validation on workload and deployment name (#291)
cloudops/cloudmc-api-docs: source/includes/stackpath/_workloads.md

Required | &nbsp;
------- | -----------

>  `name`<br/>*string* | The name of the workload. The string "workloadname-deploymentname" must not exceed 40 characters.

Reviewer: ```suggestion
 `name`<br/>*string* | The name of the workload. The deployment name for a workload must not exceed 40 characters.
```

Author: but wouldn't it be confusing for the user? they could have both the strings under 40 chars and the combination might still be invalid (>40)

Reviewer: "workloadname-deploymentname" is a bit confusing. How about - 
```suggestion
 `name`<br/>*string* | The name of the workload. The length of `<workload_name>-<deployment_name>` must not exceed 40 characters.
```

Author: yes, perfect. I'll update it

Author: a5efd65

 `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based.
 `image`<br/>*string* | Either the location of a Docker image to run as a container or the image to use for the virtual machine. If for a virtual machine, this is in the format of /[:]. If the image tag portion is omitted, 'default' is assumed which is the most recently created, ready, and non-deprecated image of that slug. A set of common images is present on the 'stackpath-edge' stack.
 `vpc`<br/>*string* | The virtual private cloud option for now supports default vpc only.
 `firstBootSshKey`<br/>*string* | If creating a VM-based workload, SSH keys are required. Multiple SSH keys can be separated by newlines `\n`.
 `specs`<br/>*string* | Specification type for resources which are allocated to each instance in a workload. Supported specifications are `SP-1 (1 vCPU, 2 GB RAM)`,`SP-2 (2 vCPU, 4 GB RAM)`,`SP-3 (2 vCPU, 8GB RAM)`,`SP-4 (4 vCPU, 16 GB RAM)`,`SP-5 (8 vCPU, 32 GB RAM)`.

>  `deploymentName`<br/>*string* | The name of the deployment. The string "workloadname-deploymentname" must not exceed 40 characters.

Reviewer: same as above

Author: a5efd65

Required | &nbsp;
------- | -----------

> `name`<br/>*string* | The name of the workload. The string "workloadname-deploymentname" must not exceed 40 characters.

Reviewer: same as above

> `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based. Can be either 'VM' or 'CONTAINER'.

Reviewer: ```suggestion
`type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based. Can be either `VM` or `CONTAINER`.
```

Similar changes can be made at other places as well.

Author: good point. 1a15407

`image`<br/>*string* | The location of a Docker image to run as a container. Only editable when `type`is equal to 'CONTAINER'.
`specs`<br/>*string* | Specification type for resources which are allocated to each instance in a workload.

> `deploymentName`<br/>*string* | The name of the deployment. The string "workloadname-deploymentname" must not exceed 40 characters.

Reviewer: same as above
